# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qemu/qemu-1.1.1-parallel-make.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix compiling with more than one make job

Compiling virtfs-proxy-helper depends on then generated config-host.h

Following error occurs without this patch
--------------------------------------------------------------------------
In file included from ./qemu-common.h:5:0,
                 from fsdev/virtfs-proxy-helper.c:23:
./compiler.h:6:25: fatal error: config-host.h: No such file or directory
--------------------------------------------------------------------------

--- qemu-1.1.1/Makefile.orig	2012-07-22 22:21:26.953999658 +0200
+++ qemu-1.1.1/Makefile	2012-07-22 22:24:13.562030679 +0200
@@ -166,6 +166,7 @@
 
 fsdev/virtfs-proxy-helper$(EXESUF): fsdev/virtfs-proxy-helper.o fsdev/virtio-9p-marshal.o oslib-posix.o $(trace-obj-y)
 fsdev/virtfs-proxy-helper$(EXESUF): LIBS += -lcap
+fsdev/virtfs-proxy-helper.o: $(GENERATED_HEADERS)
 
 qemu-img-cmds.h: $(SRC_PATH)/qemu-img-cmds.hx
 	$(call quiet-command,sh $(SRC_PATH)/scripts/hxtool -h < $< > $@,"  GEN   $@")
